.github/workflows/test-s3gw.yml

```diff
@@ -35,145 +31,54 @@ jobs:
-          sha256sum -c checksum \
-            && sudo mv yq /usr/bin/yq \
-            && sudo chmod +x /usr/bin/yq
-
-          python3 -m pip install -r s3tests/requirements.txt
-          python3 -m pip install -r ceph/qa/rgw/store/sfs/tests/requirements.txt
-
-      - name: Build Unittests
-        run: |
-          docker build \
-            --build-arg CMAKE_BUILD_TYPE=Debug \
-            --build-arg NPROC=16 \
-            --file s3gw/Dockerfile \
-            --tag s3gw-unittests \
-            --target s3gw-unittests \
-            .
-
-          docker images
-
-      - name: Run Unittests
-        run: |
-          docker run --rm s3gw-unittests
-
       - name: Build s3gw Container Image
         run: |
           docker build \
-            --build-arg CMAKE_BUILD_TYPE=Debug \
-            --build-arg NPROC=16 \
+            --build-arg CMAKE_BUILD_TYPE=RelWithDebugInfo \
+            --build-arg NPROC=32 \
             --file s3gw/Dockerfile \
             --tag s3gw \
             .
 
           docker images
 
-      - name: Run Integration tests
+      - name: s3tr
         run: |
-          set -e
           set -x
 
-          source ceph/qa/rgw/store/sfs/tests/helpers.sh
+          docker run --rm \
+            -v /var/run/docker.sock:/var/run/docker.sock \
+            -v $GITHUB_WORKSPACE/s3tr-out:/out \
+            ghcr.io/irq0/s3tr:latest \
+            run \
+            --image s3gw:latest \
+            /out/s3tr.json
 
-          mkdir -p integration/storage
-          CONTAINER=$(docker run --rm -d \
-            -p 7480:7480 \
-            -v $GITHUB_WORKSPACE/integration/storage:/data \
-            s3gw \
-              --rgw-backend-store sfs \
-              --debug-rgw 1 \
-              --rgw-lc-debug-interval 10)
-
-          wait_for_http_200 "http://127.0.0.1:7480"
-
-          echo "Running Integration Tests:"
-          cd ceph/qa/rgw/store/sfs/tests
-
-          python3 -m unittest -v test-*.py
-
-          docker kill $CONTAINER
+      - uses: actions/upload-artifact@v3
+        with:
+          name: s3tr-results
+          path: s3tr-out/s3tr.json
 
-      - name: Run smoke tests
+      - name: s3tr - new successes?
         run: |
-          set -e
           set -x
-
-          source ceph/qa/rgw/store/sfs/tests/helpers.sh
-
-          mkdir -p smoke/storage
-          CONTAINER=$(docker run --rm -d \
-            -p 7480:7480 \
-            -v $GITHUB_WORKSPACE/smoke/storage:/data \
-            s3gw \
-              --rgw-backend-store sfs \
-              --debug-rgw 1)
-
-          wait_for_http_200 "http://127.0.0.1:7480"
-
-          echo "Running Smoke Tests:"
-          cd ceph/qa/rgw/store/sfs/tests
-          ./sfs-smoke-test.sh 127.0.0.1:7480
-
-          docker kill $CONTAINER
-
-      - name: Run S3tests
+          docker run --rm \
+            -v $GITHUB_WORKSPACE/s3tr-out:/out \
+            -v $GITHUB_WORKSPACE/ceph:/ceph:ro \
+            ghcr.io/irq0/s3tr:latest \
```

Comment:
We're building this in CI now. It's available at `ghcr.io/aquarist-labs/s3tr:latest`